source/ranges.tex

```diff
@@ -8419,9 +8417,7 @@
 
     constexpr explicit zip_transform_view(F fun, Views... views);
 
-    constexpr auto begin() {
-      return @\exposid{iterator}@<false>(*this, @\exposid{zip_}@.begin());
-    }
+    constexpr auto begin() { return @\exposid{iterator}@<false>(*this, @\exposid{zip_}@.begin()); }
```

Comment:
Same here: definition on same line seems a bit over-the-top, but maybe it improves consistency.